frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/tilsagn/Tilsagnstabell.tsx

```diff
@@ -109,25 +89,16 @@ export function Tilsagnstabell({ tilsagn }: Props) {
       </Table.Header>
       <Table.Body>
         {tilsagn.map((tilsagn) => {
-          const {
-            periodeStart,
-            periodeSlutt,
-            kostnadssted,
-            beregning,
-            id,
-            besluttelse,
-            lopenummer,
-          } = tilsagn;
+          const { periodeStart, periodeSlutt, kostnadssted, beregning, id, besluttelse } = tilsagn;
           return (
             <Table.Row key={id}>
-              <Table.DataCell>{lopenummer}</Table.DataCell>
               <Table.DataCell>{formaterDato(periodeStart)}</Table.DataCell>
               <Table.DataCell>{formaterDato(periodeSlutt)}</Table.DataCell>
               <Table.DataCell>
                 {kostnadssted.navn} {kostnadssted.enhetsnummer}
               </Table.DataCell>
               <Table.DataCell>{formaterTall(beregning.belop)} kr</Table.DataCell>
-              <Table.DataCell>{tilsagnTilStatus(tilsagn, ansatt)}</Table.DataCell>
+              <Table.DataCell>{status(tilsagn, ansatt)}</Table.DataCell>
```

Comment:
Samme som andre kommentar, hvis `status()` i praksis er en react-kompoent, så kalle den via JSX-syntax i stedet for inline?